Tasks/NuGetRestoreV1/nugetinstaller.ts

```diff
@@ -135,14 +136,19 @@ async function main(): Promise<void> {
         // and check if the user picked the 'select' option to fill out the config file if needed
         if (selectOrConfig === "select" ) {
             let sources: Array<IPackageSource> = new Array<IPackageSource>();
-            let feed = tl.getInput("feed");
-            if (feed) {
-                let feedUrl:string = await nutil.getNuGetFeedRegistryUrl(packagingLocation.DefaultPackagingUri, feed, nuGetVersion, accessToken);
-                sources.push(<IPackageSource>
-                {
-                    feedName: feed,
-                    feedUri: feedUrl
-                })
+            let feed = getProjectAndFeedIdFromInputParam("feed");
+            
+            if (feed.feedId) {
+                if(feed.projectId) {
+                    tl.warning(tl.loc("UnsupportedProjectScopedFeeds"));
```

Comment:
>warning [](start = 23, length = 7)

This should be a failure, not just a warning.

Reply:
Fixed